README.md

```diff
@@ -77,6 +77,7 @@ These environment variables will override other options.
 * `GDT_DOMAIN`: Specify the base URL for live system testing. Falls back to a
```

Comment:
How does GDT_DOMAIN interact with GDT_SITEURLS? Should GDT_SITEURLS be documented here?